lib/cfg-grammar.y

```diff
@@ -892,7 +893,11 @@ block_arg
           LL_BLOCK
           {
             cfg_lexer_pop_context(lexer);
-            cfg_args_set(last_block_args, $1, $3); free($1); free($3);
+            if ($3 && cfg_lexer_is_backtick_substitution_postponed(lexer) && strchr($3, '`'))
```

Comment:
As it seems, here backtick substitution is always delayed. why do we need the check?

also the strchr() should probably be wrapped into a function that describes its intent.